scss/styles.scss

```diff
@@ -0,0 +1,28 @@
+/* ========================================================
+IMPORTS
+======================================================== */
+
+@import url("https://cdn.jsdelivr.net/npm/bootstrap-icons@1.9.1/font/bootstrap-icons.css");
+@import "_variables.scss";
+@import "_nav.scss";
+@import "_card.scss";
+@import "_button.scss";
```

Comment:
You do not import Bootstrap.scss. This is a hard requirement for this assignment

Reply:
This file? \node_modules\bootstrap\scss\bootstrap.scss

Reply:
That's correct